src/rdkafka_partition.c

```diff
@@ -1295,6 +1291,12 @@ static void rd_kafka_toppar_handle_Offset (rd_kafka_t *rk,
 		    err = RD_KAFKA_RESP_ERR__OUTDATED;
 	}
 
+        if (err != RD_KAFKA_RESP_ERR__OUTDATED) {
+            /* Parse and return Offset */
```

Comment:
each level of indent shall be 8 whitespaces

Reply:
ok! changed it.